Controller/ProfilerController.php

```diff
@@ -32,54 +29,54 @@ public function setContainer(ContainerInterface $container = null)
     /**
      * Renders the profiler panel for the given token.
      *
-     * @param string  $token          The profiler token
-     * @param string  $connectionName
-     * @param integer $query
+     * @param string $token          The profiler token
+     * @param string $connectionName
+     * @param int    $query
      *
      * @return Response A Response instance
      */
     public function explainAction($token, $connectionName, $query)
     {
-        /** @var $profiler \Symfony\Component\HttpKernel\Profiler\Profiler */
+        /** @var \Symfony\Component\HttpKernel\Profiler\Profiler $profiler */
         $profiler = $this->container->get('profiler');
         $profiler->disable();
 
         $profile = $profiler->loadProfile($token);
         $queries = $profile->getCollector('db')->getQueries();
 
-        if (!isset($queries[$connectionName][$query])) {
+        if (! isset($queries[$connectionName][$query])) {
             return new Response('This query does not exist.');
         }
 
         $query = $queries[$connectionName][$query];
-        if (!$query['explainable']) {
+        if (! $query['explainable']) {
             return new Response('This query cannot be explained.');
         }
 
-        /** @var $connection \Doctrine\DBAL\Connection */
+        /** @var \Doctrine\DBAL\Connection $connection */
         $connection = $this->container->get('doctrine')->getConnection($connectionName);
         try {
             if ($connection->getDatabasePlatform() instanceof SQLServerPlatform) {
                 $results = $this->explainSQLServerPlatform($connection, $query);
             } else {
                 $results = $this->explainOtherPlatform($connection, $query);
             }
-        } catch (\Exception $e) {
+        } catch (\Throwable $e) {
```

Comment:
Oops, this one needs to be reverted, because 5.x. :/ Sniff is already excluded though.